src/main/java/jenkins/plugins/clangscanbuild/publisher/ClangScanBuildPublisherDescriptor.java

```diff
@@ -41,10 +40,7 @@ public String getDisplayName() {
 
 	@Override
 	public boolean isApplicable( @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobType ){
-		if( !FreeStyleProject.class.isAssignableFrom( jobType ) ){
-			System.err.println( "Clang scan-build ERROR: Expected FreeStyleProject but was: " + jobType + " at Publisher Descriptor" );
-		}
-		return AbstractProject.class.isAssignableFrom( jobType );
+		return true;
```

Comment:
@tfennelly I thought the same, but @escoem is right: given the signature `Class<? extends AbstractProject> jobType` only `AbstractProject` extensions would reach this code, so returning `true` would be valid (assuming any `AbstractProject` implementation is acceptable).